components/formats-gpl/src/loci/formats/in/ScreenReader.java

```diff
@@ -78,6 +78,7 @@
   // -- Fields --
 
   private String[] plateMetadataFiles;
+  private String[] excludeReaders;
```

Comment:
This will likely invalidate the bfmemo cache and if so could not be introduced in a patch release.